draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -8548,7 +8554,8 @@ Content-Range: exampleunit 11.2-14.3/25
    selected representation's complete length.
 </t>
 <t>
-   A sender that generates a 206 response with an <x:ref>If-Range</x:ref>
+   A sender that generates a 206 response for a request with
```

Comment:
I did not see anything to suggest that If-Range is usable as a response header, so made some inference here.